CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/visual/InteractiveVKVisualProcessor.java

```diff
@@ -211,20 +199,20 @@ public VisualOperation flagBusy(boolean isBusy) {
 
     @Override
     public Vector3f convertZoomPointToDirection(final Point zoomPoint) {
+
         return new Vector3f(
                 (float) (getCanvas().getWidth() / 2.0 - zoomPoint.x),
-                (float) (getCanvas().getHeight() / 2.0 - zoomPoint.y),
+
+                (float) (zoomPoint.y - getCanvas().getHeight() / 2.0),
```

Comment:
should put brackets around ((zoomPoint.y - geCanvas().getHeight()) / 2.0) for clarity

Reply:
No because that would be a different operation, this moving the origin to the centre of the screen.